docs/data/tree-view/rich-tree-view/selection/selection.md

```diff
@@ -75,25 +75,36 @@ Use the `onItemSelectionToggle` prop if you want to react to an item selection c
 
 {{"demo": "TrackItemSelectionToggle.js"}}
 
-## Parent / children selection relationship
+## Automatic parents and children selection
```

Comment:
Basically a copy paste of the grid doc
If we modify this one, I'd be in favor of having Sam check the changes and apply the same to the grid one.